src/Prism.Core/Ioc/ContainerLocator.cs

```diff
@@ -9,16 +9,14 @@ namespace Prism.Ioc
     [EditorBrowsable(EditorBrowsableState.Never)]
     public static class ContainerLocator
     {
-        private static Lazy<IContainerExtension> _lazyContainer;
-
         private static IContainerExtension _current;
 
         /// <summary>
         /// Gets the current <see cref="IContainerExtension" />.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static IContainerExtension Current =>
-            _current ?? (_current = _lazyContainer?.Value);
+            _current;
```

Comment:
do we nee a private backing field anymore? can this just be a auto-prop now with a private setter?